server/models/messenger_models.py

```diff
@@ -0,0 +1,28 @@
+from api import db
+from sqlalchemy import ForeignKey
+from datetime import datetime
+
+
+user_in_session = db.Table(
+    'room_session_user',
+    db.Column('user_id', db.Integer, db.ForeignKey('user.id')),
+    db.Column('room_id', db.Integer, db.ForeignKey('room_session.id'))
+)
+
+
+class RoomSession(db.Model):
+    __tablename__ = "room_session"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
+    messages = db.relationship('Message', primaryjoin="RoomSession.id == Message.session", backref="room")
+    users = db.relationship('User', secondary=user_in_session, backref=db.backref("sessions"), lazy="dynamic")
+
+
+class Message(db.Model):
+    __tablename__ = "message"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
+    session = db.Column(db.Integer, ForeignKey("room_session.id"))
```

Comment:
this should be named `session_id`

keys should ideally have a suffix of `_id` to quickly identify it has a relationship to something else. in addition to follow a better naming convention, name this `room_session_id` since your model and table is called `room_session`, `RoomSession`